terragrunt/env/production/vpc/terragrunt.hcl

```
@@ -0,0 +1,3 @@
+terraform {
+    source = "../../../aws//vpc"
+}
```

Could you this below so that the root `terragrunt.hcl` gets included in the config.    What this does is go up through the parent directories of this file and take the first `terragrunt.hcl` file it finds and includes its config as well.
```suggestion
}

include {
  path = find_in_parent_folders()
}
```